### PR TITLE
validate package name in swift build --init

### PR DESCRIPTION
**Pull request by kostiakoval**
Fix https://bugs.swift.org/browse/SR-1224

### CODE REVIEW

**Comment by mxcl** [comment]
No bangs please, lets make `c99name` a real property and push the throw to the initializer.

**Comment by kostiakoval** [comment]
@mxcl Review plz and merge if you like it :)

**Comment by mxcl** [comment]
LGTM

@swfit-ci please test and merge

**Comment by mxcl** [comment]
@swfit-ci please test and merge

**Comment by mxcl** [comment]
@swift-ci Please test and merge

**Comment by mxcl** [comment]
Conflicts.

**Comment by kostiakoval** [comment]
Solving it

**Comment by kostiakoval** [comment]
@mxcl Done, please merge

**Comment by mxcl** [comment]
@swift-ci Please test and merge

File: Tests/Transmute/ModuleDependencyTests.swift [hunk 1]
```diff
@@ -12,128 +12,147 @@
 import class PackageType.Module
 import XCTest
 
+func testModules(file: StaticString = #file, line: UInt = #line, body: () throws -> Void) {
```
Comment (mxcl) [inline Tests/Transmute/ModuleDependencyTests.swift]: For future reference, you can just change XCTestCase functions to have `throws` now.

**Pull request merged**